--- openwrt/files/update-l1pro-openwrt.sh	2021-04-01 21:12:33.000000000 +0800
+++ mk_openwrt_src_20210401/openwrt/files/update-l1pro-openwrt.sh	2021-04-18 08:52:53.692974300 +0800
@@ -201,7 +201,11 @@
 	fi
 	ln -sf /mnt/mmcblk0p4/AdGuardHome /mnt/${NEW_ROOT_NAME}/usr/bin/AdGuardHome
 fi
-
+cp /etc/crontabs/root /mnt/${NEW_ROOT_NAME}/etc/crontabs/
+cp -rf /root/* /mnt/${NEW_ROOT_NAME}/root/
+cp -rf /www/ACLWeb /mnt/${NEW_ROOT_NAME}/www/
+cp -rf /www/Sub-Web /mnt/${NEW_ROOT_NAME}/www/
+cp -rf /www/luci-static/argon/background /mnt/${NEW_ROOT_NAME}/www/luci-static/argon/
 sync
 echo "copy done"
 echo
